crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -79,7 +80,15 @@ impl JsonRpcHandler {
         _block_id: BlockId,
         _contract_address: ContractAddressHex,
     ) -> RpcResult<ClassHashHex> {
-        Err(error::ApiError::ContractNotFound)
+        let parsed_address = _contract_address.0.try_into().unwrap(); // TODO better error handling
```

Comment:
argument names with `_` are used for arguments that aren't used in function in rust. All arguments that are used within the function shall be without `_` in front

Reply:
Addressed in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/50/commits/7e468c61b374117e62f877894b80f76ae2340c77